api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java

```diff
@@ -361,7 +361,7 @@ private CohortDefinition getNumberOfPatientsInFichaClinicaWithViralLoadResult()
    *
    * <p>It was duplicated here because the spec pointed to an incompatible query
    */
-  private CohortDefinition getPatientsTransferredOut() {
+  public CohortDefinition getPatientsTransferredOut() {
```

Comment:
Does it need to be public?

Reply:
No, was done for live testing purposes. Reverted